internal/restic/node_xattr_darwin.go

```diff
@@ -0,0 +1,15 @@
+package restic
```

Comment:
This file is the exact same as the Linux version, but we haven't established that the problem occurs on MacOS.

Reply:
ok, node_xattr_darwin.go have been removed